src/docs/navigation.js

```diff
@@ -14,69 +14,118 @@ export const GUIDES = [
     title: "Education and Use cases",
     description: "Examples and use cases for Santiment metrics",
     icon: Education,
-    articles: ["Timing market volatility with Token Age Consumed","Investigate Speculative Activity with Santiment’s Exchange Metrics", "How To Spot tops with Price - Network Activity divergences","Price to Daily Addresses Divergence - Primer on On-chain Trading Strategies"]
+    articles: [
+      "Timing market volatility with Token Age Consumed",
+      "Investigate Speculative Activity with Santiment’s Exchange Metrics",
+      "How To Spot tops with Price - Network Activity divergences",
+      "Price to Daily Addresses Divergence - Primer on On-chain Trading Strategies",
+    ],
   },
   {
     title: "Sanbase",
     icon: Sanbase,
     description: "Our crypto analytics & behavior platform",
-    articles: ["Assets page", "Insights", "Sonar page", "Emerging trends page", "Social trends search", "Account settings", "Logging into Sanbase","Requesting display new project"],
-
+    articles: [
+      "Assets page",
+      "Insights",
+      "Sonar page",
+      "Emerging trends page",
+      "Social trends search",
+      "Account settings",
+      "Logging into Sanbase",
+      "Requesting display new project",
+    ],
   },
   {
     title: "SanAPI",
     icon: SanAPI,
-    description: "Our API, available metrics & queries",
-    articles: ["Exploring", "Available queries", "Available metrics"],
+    description: "Santiment API, Available Metrics & Queries",
+    articles: ["Metrics", "Queries"],
   },
   {
     title: "Sansheets",
     icon: Sansheets,
     description: "Our Spreadsheets plugin for Santiment data",
-    articles: ["Setting up", "Adding an API key", "Functions", "Pro Templates"]
+    articles: ["Setting Up", "Adding an API key", "Functions", "PRO Templates"],
   },
   {
     title: "Sandata",
     icon: Sandata,
     description: "Our advanced crypto dashboards",
-    articles: ["Logging into Sandata", "Navigation", "The erc 20 overview page"],
+    articles: ["Logging into Sandata", "Navigation", "ERC20 Overview Page"],
```

Comment:
If you rename here - you need also rename related folder name. Otherwise page link will be unreachable from sidebar

Reply:
resolved